changelogs/current.yaml

```diff
@@ -204,6 +204,9 @@ new_features:
 - area: access_log
   change: |
     updated command operator ``%GRPC_STATUS%`` to suppoprt the snake case.
+- area: redis
+  change: |
+    added support for redis transactions.
```

Comment:
Can you modify https://www.envoyproxy.io/docs/envoy/latest/intro/arch_overview/other_protocols/redis to talk about the transaction support briefly?

Reply:
Done.